src/webview-contract/webviewDefinitions/createFleet.ts

```diff
@@ -0,0 +1,59 @@
+import { WebviewDefinition } from "../webviewTypes";
+
+export interface InitialState {
+    subscriptionId: string;
+    subscriptionName: string;
+}
+
+export interface ResourceGroup {
+    name: string;
+    location: string;
+}
+
+export enum ProgressEventType {
+    InProgress,
+    Cancelled,
+    Failed,
+    Success,
+}
+
+export type CreatedFleet = {
+    portalUrl: string;
+};
+
+export interface CreateFleetParams {
```

Comment:
isn't this the create fleet argument list?
should this be used as an argument to the createFleet func?